client/chainlink.go

```diff
@@ -230,6 +231,14 @@ func (c *chainlink) ReadETHKeys() (*ETHKeys, error) {
 	return ethKeys, err
 }
 
+// ReadVRFKeys reads all VRF keys from the Chainlink node
+func (c *chainlink) ReadVRFKeys() (*VRFKeys, error) {
```

Comment:
Need to merge VRF key generation in compose first https://github.com/smartcontractkit/chainlink-node-compose/pull/10